### PR TITLE
introduce command_custom option

### PR DESCRIPTION
**Pull request by thyseus**

### CODE REVIEW

File: src/Console/Scheduling/Schedule.php [hunk 1]
```diff
@@ -5,77 +5,80 @@
...
+	public function dueEvents($app)
+	{
+		if ($this->isScheduleAdded) {
+			return parent::dueEvents($app);
+		}
+		$scheduleService = app(ScheduleService::class);
+		$schedules = $scheduleService->getActives();
+
+		foreach ($schedules as $schedule) {
+			$command = $schedule->command;
```
[the remark this answers is not in the document]
Comment (thyseus) [reply src/Console/Scheduling/Schedule.php]: nice ! will fix.

[the remark this answers is not in the document]
Comment (thyseus) [reply src/Console/Scheduling/Schedule.php]: Ah, eyes up - executing a custom command needs to be called by `exec()`, not by `command()` ...

File: migrations/2021_03_16_171148_add_custom_command.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+use Illuminate\Support\Facades\Schema;
+
+class AddCustomCommand extends Migration
+{
+	/**
+	 * Run the migrations.
+	 */
+	public function up()
+	{
+		Schema::table('schedules', function (Blueprint $table) {
```
Comment (robersonfaria) [inline migrations/2021_03_16_171148_add_custom_command.php:14]: Here in the migration, the name of the table must use the field of the `config/database-schedule.php` configuration file

File: migrations/2021_03_16_171148_add_custom_command.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			$table->string('command_custom')->after('command')->nullable()->default(null);
+		});
+	}
+
+	/**
+	 * Reverse the migrations.
+	 */
+	public function down()
+	{
+		Schema::table('schedules', function (Blueprint $table) {
```
Comment (robersonfaria) [inline migrations/2021_03_16_171148_add_custom_command.php:24]: here the same thing

File: resources/views/form.blade.php [hunk 4]
```diff
@@ -72,6 +72,22 @@ class="form-control @error('command') is-invalid @enderror"
         <code>{{ trans('schedule::schedule.messages.attention-type-function') }}</code>
     </div>
 
+    <div v-if="form.command == 'custom'" class="form-group">
```
Comment (robersonfaria) [inline resources/views/form.blade.php:75]: I believe that the options in the command field lacked the option `custom`.
Or did I misunderstand how to select a custom field?

**Review by robersonfaria — CHANGES_REQUESTED** [review]
Your contribution is very good, thank you very much.

I just made a few observations, if you can change it or if you think we should do something different, I’m open to talk about it.

Thanks again for the contribution.

**Review by robersonfaria — CHANGES_REQUESTED** [review]
for now these are the considerations.

Thank you very much

File: src/Console/Scheduling/Schedule.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (thyseus) [reply src/Console/Scheduling/Schedule.php]: Thats clean code! Integrated it.

**Review by robersonfaria — APPROVED** [review]
Added the possibility to execute custom commands.

**Pull request merged**